sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -744,8 +744,10 @@ public abstract partial class TokenCredential
         private readonly int _dummyPrimitive;
         public TokenRequestContext(string[] scopes, string? parentRequestId) { throw null; }
         public TokenRequestContext(string[] scopes, string? parentRequestId, string? claims) { throw null; }
-        public TokenRequestContext(string[] scopes, string? parentRequestId = null, string? claims = null, string? tenantId = null) { throw null; }
+        public TokenRequestContext(string[] scopes, string? parentRequestId, string? claims, string? tenantId) { throw null; }
+        public TokenRequestContext(string[] scopes, string? parentRequestId = null, string? claims = null, string? tenantId = null, bool enableCae = false) { throw null; }
         public string? Claims { get { throw null; } }
+        public bool EnableCae { get { throw null; } }
```

Comment:
this sounds like a method name. Bool property name should probably be something like IsCaeEnabled.